src/stories/Quiz/tests/index.stories.tsx

```diff
@@ -20,33 +21,25 @@ export default {
         type: 'code',
       },
     },
+    controls: docsControls,
   },
 };
 
-const resultsPageOptions = {
-  onQuizResultClick: (result, position) => {
-    console.log('Click result');
-    console.dir(result, position);
-  },
-  onQuizResultsLoaded: (results) => {
-    console.log('Loaded results');
-    console.dir(results);
-  },
-  onAddToCartClick: (item) => {
-    console.log('Add to cart');
-    console.dir(item);
-  },
-  resultCardRegularPriceKey: 'price',
-  resultCardSalePriceKey: 'salePrice',
-};
-
 const sessionStateOptions = {
   showSessionModal: false,
 };
 
 // eslint-disable-next-line storybook/prefer-pascal-case
 export const e2eInteractionTest = ComponentTemplate.bind({});
-e2eInteractionTest.args = { apiKey, quizId, resultsPageOptions, sessionStateOptions };
+e2eInteractionTest.args = {
```

Comment:
Hopefully, I'm leaving this comment in the right place.

Would it be possible to move the `Full Quiz e2e tests` story to the end? Also, moving Questions and Results right before it, but also towards the end.

Please reference
![Screenshot 2023-10-05 at 5 32 45 PM](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/7194266/b211e7b1-1dfd-4cf9-b9f9-095c7db3cd10)
 my poor attached drawing.